### PR TITLE
adds starter landing/home page

### PR DESCRIPTION
**Pull request by dawnkelly09**
Adds a starter landing/home page for Polkadot Ecosystem Docs

-> I did run this through Grammarly

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -1,8 +1,32 @@
...
-  - navigation
   - feedback
-  - toc
 ---
+
+# Polkadot Ecosystem Documentation
+
+Welcome to the documentation hub for Polkadot, a scalable, interoperable, and secure network empowering the next generation of web builders. This site aims to serve as a central knowledge exchange for teams across the Polkadot ecosystem. 
+
+Here, you will find detailed Polkadot information, including key features, tools, and SDKs for parachain and appchain builders. These resources provide practical benefits and best practices for working with smart contracts, pallets, and Substrate and offer guidance for those building tooling and running infrastructure.
```
Comment (eshaben) [inline docs/index.md:12]: Oooh thisis good! I'm thinking that we could sort of fill the page out a bit more if we take this second paragraph and create a section that elaborates on the resources and documentation that will be covered on the developer hub. 

What do you think about adding a section like this:


## Comprehensive Developer Resources

Here's what you'll be able to find in the Developer Hub:

- A deep dive into Polkadot - Get familiar with Polkadot's architecture, inner workings, and core technologies, including Agile Coretime, XCM, bridges, governance, and more
- Resources for parachain development - Learn how to create, test, and deploy a parachain from scratch using Substrate and explore how to build and customize your runtime, including adding pallets and implementing custom logic
- Parachain tooling - Access specialized tools and frameworks to streamline parachain development, from launch to maintenance
- Documentation on smart contract development - Find resources and tutorials on developing, testing, and deploying smart contracts using Solidity and ink! on Polkadot's EVM and other supported EVM environments in the ecosystem
- Ecosystem tooling - Discover a suite of powerful tools, including SDKs, APIs, and other resources, to build scalable, interoperable applications on Polkadot
- Information on running infrastructure - Infrastructure providers can find detailed information on operating and maintaining validator nodes, including block production, transaction validation, and ensuring network security

These resources provide practical benefits and best practices for working with smart contracts, pallets, and Substrate and offer guidance for those building tooling and running infrastructure.

File: docs/index.md [hunk 2]
```diff
@@ -1,8 +1,32 @@
...
+
+## What is Polkadot? {: #what-is-polkadot }
```
Comment (eshaben) [inline docs/index.md:14]: I think we can get rid of the What is Polkadot and Get Started sections for now

**Review by eshaben — COMMENTED** [review]
Nice! I think we're getting some ideas cooking!

@albertov19 figured I would add you as a reviewer in case you want to check it out or have any additional ideas

File: docs/index.md [hunk 3]
```diff
@@ -1,8 +1,32 @@
...
+description: Explore the information, tools, and tutorials needed to start building on top of Polkadot, a blockchain network protocol that provides parachains with shared security and interoperability using XCM. 
 hide:
-  - navigation
   - feedback
-  - toc
 ---
+
+# Polkadot Ecosystem Documentation
+
+Welcome to the documentation hub for Polkadot, a scalable, interoperable, and secure network empowering the next generation of web builders. This site aims to serve as a central knowledge exchange for teams across the Polkadot ecosystem.
```
Comment (eshaben) [inline docs/index.md:10]:
```suggestion
Welcome to the documentation hub for Polkadot, a scalable, interoperable, and secure network empowering the next generation of web builders. Whether you're creating parachains, developing interoperable tools, or building decentralized applications, this site aims to serve as a central knowledge exchange for developers across the Polkadot ecosystem.
```

**Review by albertov19 — CHANGES_REQUESTED** [review]
Looks like a good starting point. I pushed a commit with some logo and minor modifications. @dawnkelly09 check it out so you learn about that in our Docs framework :) 

I included a new flag in the yml config file to ignore the absolute link warning -> https://www.mkdocs.org/user-guide/configuration/#validation (cc @eshaben )

I removed the footer but we should incorporate a left-side menu navigation to make the docs we create accessible to people. I did not look into how to adding that (it was by default before @eshaben )

File: docs/index.md [hunk 4]
```diff
@@ -1,8 +1,30 @@
...
-  - toc
+title: Polkadot Ecosystem Docs
+description: Explore the information, tools, and tutorials needed to start building on top of Polkadot, a blockchain network protocol that provides parachains with shared security and interoperability using XCM. 
 ---
+
+# Polkadot Ecosystem Documentation
+
+Welcome to a draft documentation site for ecosystem-related things on Polkadot, a scalable, interoperable, and secure network empowering the next generation of web builders. This site aims to serve as a central knowledge exchange for teams across the Polkadot ecosystem. 
+
+Here, you will find detailed Polkadot information, including key features, tools, and SDKs for parachain and appchain builders. These resources provide practical benefits and best practices for working with smart contracts, pallets, and Substrate and offer guidance for those building tooling and running infrastructure.
```
Comment (dawnkelly09) [inline docs/index.md:10]:
```suggestion
Here's what you'll be able to find in the Developer Hub:

- A deep dive into Polkadot - Get familiar with Polkadot's architecture, inner workings, and core technologies, including Agile Coretime, XCM, bridges, governance, and more
- Resources for parachain development - Learn how to create, test, and deploy a parachain from scratch using Substrate and explore how to build and customize your runtime, including adding pallets and implementing custom logic
- Parachain tooling - Access specialized tools and frameworks to streamline parachain development, from launch to maintenance
Documentation on smart contract development - Find resources and tutorials on developing, testing, and deploying smart contracts using Solidity and ink! on Polkadot's EVM and other supported EVM environments in the ecosystem
- Ecosystem tooling - Discover a suite of powerful tools, including SDKs, APIs, and other resources, to build scalable, interoperable applications on Polkadot
- Information on running infrastructure - Infrastructure providers can find detailed information on operating and maintaining validator nodes, including block production, transaction validation, and ensuring network security

These resources provide practical benefits and best practices for working with smart contracts, pallets, and Substrate and offer guidance for those building tooling and running infrastructure.
```

File: docs/index.md [hunk 5]
```diff
@@ -1,8 +1,30 @@
...
+
+## What is Polkadot? {: #what-is-polkadot }
```
Comment (dawnkelly09) [inline docs/index.md:12]:
```suggestion

```

File: docs/index.md [hunk 6]
```diff
@@ -1,8 +1,30 @@
...
+
+Polkadot is a blockchain network protocol that provides shared security among all parachains and allows all connected tasks to interoperate using XCM. The XCM messaging format will enable blockchains to pass any data to other chains or consensus mechanisms, allowing for secure interoperability and scaling of connected chains. This foundation lets each parachain specialize towards a specific focus and optimize toward that goal without the risk of compromising user security or cross-chain capabilities.
```
Comment (dawnkelly09) [inline docs/index.md:14]:
```suggestion

```

File: docs/index.md [hunk 7]
```diff
@@ -1,8 +1,30 @@
...
+
+## Get Started with Polkadot {: #get-started-with-polkadot }
```
Comment (dawnkelly09) [inline docs/index.md:16]:
```suggestion

```

**Review by dawnkelly09 — COMMENTED** [review]
Updated suggestions to match comments from reviewers.

File: docs/index.md [hunk 8]
```diff
@@ -1,8 +1,32 @@
...
+
+### Build a Parachain {: #build-a-parachain }
+
+- [Paseo Testnet](/paseo-testnet/onboarding) - deploy a custom parachain to the Paseo Testnet
```
[the remark this answers is not in the document]
Comment (eshaben) [reply docs/index.md:26]: Yes, we did. But I think all of these sections can be removed for right now until we get some more content added

File: docs/index.md [hunk 9]
```diff
@@ -1,8 +1,36 @@
...
+Documentation on smart contract development - Find resources and tutorials on developing, testing, and deploying smart contracts using Solidity and ink! on Polkadot's EVM and other supported EVM environments in the ecosystem
+- Ecosystem tooling - Discover a suite of powerful tools, including SDKs, APIs, and other resources, to build scalable, interoperable applications on Polkadot
+- Information on running infrastructure - Infrastructure providers can find detailed information on operating and maintaining validator nodes, including block production, transaction validation, and ensuring network security
+
+These resources provide practical benefits and best practices for working with smart contracts, pallets, and Substrate and offer guidance for those building tooling and running infrastructure.
+
+
+
+
+### Build a Parachain {: #build-a-parachain }
```
Comment (dawnkelly09) [inline docs/index.md:24]:
```suggestion

```

File: docs/index.md [hunk 10]
```diff
@@ -1,8 +1,36 @@
...
+
+- [Paseo Testnet](/paseo-testnet/onboarding) - deploy a custom parachain to the Paseo Testnet
```
Comment (dawnkelly09) [inline docs/index.md:26]:
```suggestion

```

File: docs/index.md [hunk 11]
```diff
@@ -1,8 +1,36 @@
...
+
+### Develop Smart Contracts {: develop-smart-contracts}
```
Comment (dawnkelly09) [inline docs/index.md:28]:
```suggestion

```

File: docs/index.md [hunk 12]
```diff
@@ -1,8 +1,36 @@
...
+
+
+### Create Tooling
```
Comment (dawnkelly09) [inline docs/index.md:31]:
```suggestion

```

File: docs/index.md [hunk 13]
```diff
@@ -1,8 +1,36 @@
...
+
+- [Paseo Testnet](/paseo-testnet/onboarding) - deploy a custom parachain to the Paseo Testnet
+
+### Develop Smart Contracts {: develop-smart-contracts}
+
+
+### Create Tooling 
+
+
+### Validate and Produce Blocks
```
Comment (dawnkelly09) [inline docs/index.md:34]:
```suggestion
```

File: docs/index.md [hunk 14]
```diff
@@ -1,8 +1,36 @@
...
+<!--TODO: Rename this? Need a name for this Infrastructure section-->
```
Comment (dawnkelly09) [inline docs/index.md:35]:
```suggestion

```

**Review by eshaben — APPROVED** [review]
LGTM! 🚀

**Pull request merged**